### PR TITLE
Port type-of

### PR DESCRIPTION
**Pull request by shaleh**

### CODE REVIEW

**Comment by shaleh** [comment]
Retriggering a build

**Comment by shaleh** [comment]
One more time...

**Comment by shaleh** [comment]
And again....

File: rust_src/src/data.rs [hunk 1]
```diff
@@ -54,4 +59,72 @@ pub fn indirect_function_lisp(object: LispObject, _noerror: LispObject) -> LispO
     return result;
 }
 
+/// a symbol representing the type of OBJECT.
+/// The symbol returned names the object's basic type;
```
Comment (brotzeit) [inline rust_src/src/data.rs]: It should be "Return a symbol".

**Pull request merged**